lib/oban/cron/expression.ex

```diff
@@ -110,12 +110,23 @@ defmodule Oban.Cron.Expression do
 
   defp parse_part(part, range) do
     cond do
-      part == "*" -> range
-      part =~ ~r/^\d+$/ -> parse_literal(part)
-      part =~ ~r/^\*\/[1-9]\d?$/ -> parse_step(part, range)
-      part =~ ~r/^\d+(\-\d+)?\/[1-9]\d?$/ -> parse_range_step(part, range)
-      part =~ ~r/^\d+\-\d+$/ -> parse_range(part, range)
-      true -> raise ArgumentError, "unrecognized cron expression: #{part}"
+      part == "*" ->
```

Comment:
The extra space here makes it more readable. Are there any other changes to the conditions? My visual regex diffing can't spot anything 🙂

Reply:
I wasn't sure about pushing that change because it was done automatically by my editor, but after seeing the result, it was more readable, as you also noticed; I'm glad you accepted the change.

> Are there any other changes to the conditions? 

No, the 🥩 of the fix was in the `parse_range/2` helper.